lib/methods/export.js

```diff
@@ -33,9 +33,11 @@ module.exports = self => {
 
       const hasRelatedTypes = !!relatedTypes.length;
 
-      const docs = (await self.getDocs(req, ids, hasRelatedTypes, manager, reporting))
+      const docs = (await self.getDocs(req, ids, manager, reporting))
```

Comment:
?

Reply:
Since we do not get relationships whatever if `export related docs` has been checked or no, we don't need this param anymore.